### PR TITLE
added national parks dataset, three prototype prompts and tests

### PR DESCRIPTION
**Pull request by hannahhch**

### CODE REVIEW

File: test/prototype-test.js [hunk 1]
```diff
@@ -233,7 +234,46 @@ describe("PROTOTYPES", () => {
         capacity: 32
       }])
     })
-  }), describe("Brewery Prompts", () => {
+  }),
+  describe("National Park Prompts", () => {
+    it("getParkVisitList", () => {
+      const e = nationalParksPrompts.getParkVisitList();
+      expect(e).to.deep.equal({
+        parksToVist: [ 'Yellowstone', 'Glacier', 'Everglades' ],
```
Comment (khalidwilliams) [inline test/prototype-test.js]: Should this be `parksToVisit`?

File: prototypes/datasets/nationalParks.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+const nationalParks = [
```
Comment (khalidwilliams) [inline prototypes/datasets/nationalParks.js:1]: This is a nice dataset! Thanks for putting this together

File: prototypes/index.js [hunk 3]
```diff
@@ -297,6 +298,71 @@ const classPrompts = {
...
+    //}
+
+    const result = 'REPLACE WITH YOUR RESULT HERE';
+    return result;
+
+    // Annotation:
+    // Write your annotation here as a comment
+  },
+
+  getParkInEachState() {
```
Comment (khalidwilliams) [inline prototypes/index.js:326]: Love this prompt, think it will reinforce some use cases for bracket notation students don't often see

**Review by khalidwilliams — CHANGES_REQUESTED** [review]
Thanks for putting this together! There's one issue in a test that I think may be a spelling error (`prototype-test.js line 242`), but otherwise this looks great!

File: prototypes/index.js [hunk 4]
```diff
@@ -297,6 +298,71 @@ const classPrompts = {
...
+
+
+// DATASET: nationalParks from ./datasets/nationalParks
+
+const nationalParksPrompts = {
+  getParkVisitList() {
+    /// Return an object containing the names of which parks I need to visit
+    // and the ones I have already visited eg:
+    // {
+    //   parksToVist: ["Yellowstone", "Glacier", "Everglades"],
```
Comment (khalidwilliams) [inline prototypes/index.js]: I didn't realize it was `Vist` in both files -- was that intentional?

Comment (hannahhch) [reply prototypes/index.js]: Nope, I just can't spell visit! :)

**Pull request merged**